api/src/Controller/Member/GetStatus.php

```diff
@@ -65,6 +65,7 @@ public function buildResource(Request $request, Response $response, $args): arra
         }
         $data = $data['users'][0];
         if (!array_key_exists('id', $data)) {
+            $error = 'User ID not found';
```

Comment:
Is this relevant to the rest of the changes or is this just something you found and wanted to fix as well?

Reply:
Since $error is otherwise undefined, the rest of this clause is invalid without this.

Reply:
so a bug fix!  Thank!  Feels like a new commit maybe a new PR, but honestly for a small commit like this just make the one line fix, give it a good git commit description and push it.